docs/sources/setup-grafana/configure-security/configure-authentication/jwt/index.md

```diff
@@ -36,7 +36,7 @@ header_name = X-JWT-Assertion
 
 ## Configure login claim
 
-To identify the user, some of the claims needs to be selected as a login info. You could specify a claim that contains either a username or an email of the Grafana user.
+To identify the user, some of the claims needs to be selected as a login info. The subject claim called `"sub"` is mandatory and needs to identify the principal that is the subject of the JWT. 
```

Comment:
@ptz-xyt you have empty whitespace at the end of the sentence. This is breaking the build. Here's a code suggestion.

```suggestion
To identify the user, some of the claims needs to be selected as a login info. The subject claim called `"sub"` is mandatory and needs to identify the principal that is the subject of the JWT.
```